src/containers/ToysContainer.js

```diff
@@ -28,7 +22,7 @@ const ToysContainer = () => {
         />
         <Route
           path="/toys/:id"
-          render={(routeProps) => <DisplayToy {...routeProps} toys={toys} />}
+          render={(routeProps) => <DisplayToy {...routeProps}  />}
```

Comment:
@Halezmarie Wanna look into useHistory() hook?  Might be able to avoid having to pass `routeProps` if you use that hook.